pkg/doc/signature/verifier/public_key_verifier_test.go

```diff
@@ -374,3 +374,38 @@ type testSignatureVerifier struct {
 func (v testSignatureVerifier) Verify(*PublicKey, []byte, []byte) error {
 	return v.verifyResult
 }
+
+func newCryptoSigner(keyType kmsapi.KeyType) (signature.Signer, error) {
+	p := mockkms.NewProviderForKMS(storage.NewMockStoreProvider(), &noop.NoLock{})
+	localKMS, err := localkms.New("local-lock://custom/master/key/", p)
+
+	if err != nil {
+		return nil, err
+	}
+
+	tinkCrypto, err := tinkcrypto.New()
+	if err != nil {
+		return nil, err
+	}
+
+	return signature.NewCryptoSigner(tinkCrypto, localKMS, keyType)
+}
+
+func mapECCurveToKeyType(curve elliptic.Curve) kmsapi.KeyType {
+	switch curve {
+	case elliptic.P256():
+		return kmsapi.ECDSAP256TypeIEEEP1363
+
+	case elliptic.P384():
+		return kmsapi.ECDSAP384TypeIEEEP1363
+
+	case elliptic.P521():
+		return kmsapi.ECDSAP521TypeIEEEP1363
+
+	case btcec.S256():
+		return kmsapi.ECDSASecp256k1TypeIEEEP1363
+
+	default:
+		panic("unsupported curve")
+	}
+}
```

Comment:
same comment about moving this to an internal utility function as it is duplicated here again.

Reply:
Done!